packages/nextjs/components/onchain-impact-dashboard/projectTotalsGraph/projectTotalsGraph.tsx

```diff
@@ -143,37 +151,35 @@ export const ProjectTotalsGraph = ({
                 </div>
               )}
             </div>
-
-            <ResponsiveContainer width="100%" className={"absolute top-14"}>
+            <div>
               {isEmpty ? (
                 <div className="flex justify-center items-center w-full h-[300px] text-neutral-content">
                   No data found
                 </div>
               ) : (
-                <AreaChart data={totalsRecord} margin={{ top: 20, right: -16, bottom: 45, left: 2 }}>
-                  <Area
-                    key={metricToWork.name}
-                    type="monotone"
-                    dataKey={metricToWork.name}
-                    stroke={stringToColor(metricToWork.label ?? "")}
-                    fill={stringToColor(metricToWork.label ?? "")}
-                  />
-                  <CartesianGrid strokeDasharray="3 3" />
-                  <XAxis tick={false} fontSize={10} hide={false} dataKey="date" />
-                  <YAxis
-                    fontSize={10}
-                    type={"number"}
-                    domain={[minValue - buffer, maxValue + buffer]}
-                    hide={false}
-                    allowDataOverflow={true}
-                    orientation="right"
-                    tickFormatter={v => formatNumber(v, 0)}
-                    ticks={[minValue - buffer, maxValue + buffer]}
-                  />
-                  <Tooltip content={<CustomTooltip />} />
-                </AreaChart>
+                <HighchartsReact
+                  highcharts={Highcharts}
+                  options={{
+                    ...options,
+                    yAxis: {
+                      title: {
+                        text: null,
+                      },
+                      min: minValue - buffer,
+                      max: maxValue + buffer,
+                    },
+                    series: [
+                      {
+                        name: metricToWork.label,
+                        data: newValues,
+                        color: stringToColor(metricToWork.label || ""),
+                      },
+                    ],
```

Comment:
We'll also need to add an option to format the tooltip date so that the weekday name isn't included
```js
tooltip: {
  xDateFormat: "%b %e, %Y",
},
```

Reply:
done